aiobotocore/client.py

```diff
@@ -183,6 +186,17 @@ def get_paginator(self, operation_name):
                 self._cache['page_config'][actual_operation_name])
             return paginator
 
+    if PY_35:
+        @asyncio.coroutine
+        def __aenter__(self):
```

Comment:
this is so you can do:
```python
session = aiobotocore.get_session()
async with session.create_client() as client:
    # do work
```

this way the TCPConnector is closed when you're done using the client and you don't need to call client.close()